src/parts/Actions.js

```diff
@@ -0,0 +1,35 @@
+import cx from 'classnames'
```

Comment:
All files in `src/parts/*` are the output of the component part script.  You can ignore these since that was replaced by the component factory function.